### PR TITLE
Use native DOM APIs instead of jQuery.

### PR DESCRIPTION
**Pull request by nybblr**
Headsup, I removed the jQuery script tag since we are no longer using it. @tracymusiker do you have WIP pull requests that are using jQuery? If so, let's pair!

@tracymusiker Code review, please! :eyes:

### CODE REVIEW

**Comment by TRACYMUSIKER** [comment]
watched this happen in front of my eyes, so code is 💯

**Comment by somelinuxguy** [comment]
Can we include more cowbell, before the deployment?

File: index.html [hunk 1]
```diff
@@ -9,7 +9,6 @@
 <body>
     <nav class="navbar"></nav>
     <ul class="image-list"></ul>
-    <script src="https://unpkg.com/jquery@3.3.1/dist/jquery.js"></script>
     <script src="main.js"></script>
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (nybblr) [inline index.html:14]: This looks like a spurious change, let's get rid of this change.

**Comment by TRACYMUSIKER** [comment]
👍

**Pull request merged**